src/api/application/application.ts

```diff
@@ -175,6 +175,17 @@ export default class ApplicationModule extends Base {
         return app;
     }
 
+    /**
+     * Runs an array of applications from manifest.
```

Comment:
Is this synchronous or asynchronous?

Reply:
async

Reply:
Sorry, I meant "this comment should specify whether the function is sync or async"

Reply:
Ahhhhh changed!